### PR TITLE
Add installation instructions to docs on render

### PR DESCRIPTION
**Pull request by nstarman**
Add package installation to the rendered docs.

@pllim, ready for review.

Signed-off-by: Nathaniel Starkman <nstarkman@protonmail.com>

### CODE REVIEW

**Comment by nstarman** [comment]
Need to add to changelog.

File: {{ cookiecutter.package_name }}/docs/index.rst [hunk 1]
```diff
@@ -6,6 +6,7 @@ This is the documentation for {{ cookiecutter.package_name }}.
 .. toctree::
   :maxdepth: 2
 
+  documentation/installation
```
Comment (pllim) [inline {{ cookiecutter.package_name }}/docs/index.rst:9]: "documentation" dir under "docs" seems a little redundant.
```suggestion
  installation
```

File: {{ cookiecutter.package_name }}/docs/documentation/installation.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+.. BADGES
+
+.. |PyPI| image:: https://badge.fury.io/py/{{ cookiecutter.package_name }}.svg
+   :target: https://badge.fury.io/py/{{ cookiecutter.package_name }}
+
+.. |PyPI Format| image:: https://img.shields.io/pypi/format/{{ cookiecutter.package_name }}?style=flat
+   :alt: PyPI - Format
+
+.. |Code Size| image:: https://img.shields.io/github/languages/code-size/{{ cookiecutter.github_project }}?style=flat
+   :alt: GitHub code size in bytes
```
Comment (pllim) [inline {{ cookiecutter.package_name }}/docs/documentation/installation.rst:54]: I would say we omit the badges. Not everyone wants them.

File: {{ cookiecutter.package_name }}/docs/documentation/installation.rst [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+.. code-block:: bash
+   :caption: setup.py
+   :name: setup-py
+
+   git clone https://github.com/{{ cookiecutter.github_project }}.git
+
+   cd {{ cookiecutter.package_name }}
+
+   python setup.py install
```
Comment (pllim) [inline {{ cookiecutter.package_name }}/docs/documentation/installation.rst:39]:
```suggestion
   pip install .
```

File: {{ cookiecutter.package_name }}/docs/documentation/installation.rst [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+From source
+***********
+
+To install the latest version, or from a branch, *{{ cookiecutter.package_name }}* can be installed from `source <https://github.com/{{ cookiecutter.github_project }}.git>`_.
+
+From the command line.
+
+.. code-block:: bash
+   :caption: setup.py
+   :name: setup-py
```
Comment (pllim) [inline {{ cookiecutter.package_name }}/docs/documentation/installation.rst:33]:
```suggestion
```

File: {{ cookiecutter.package_name }}/docs/documentation/installation.rst [hunk 5]
```diff
@@ -0,0 +1,54 @@
+.. _{{ cookiecutter.module_name }}-installation:
+
+============
+Installation
+============
+
+|Code Size|
+
+**********
+With `pip`
```
Comment (pllim) [inline {{ cookiecutter.package_name }}/docs/documentation/installation.rst:10]: Not everyone put their packages on PyPI.

**Review by pllim — COMMENTED** [review]
In general, I am not sure if this is necessary. A very short but concise "install" section under README might be sufficient for a start. People can create their RST on their own if they want to get fancy later.

File: {{ cookiecutter.package_name }}/docs/index.rst (hunk 1 above)
Comment (nstarman) [reply {{ cookiecutter.package_name }}/docs/index.rst:9]: I've seen this in a few packages, to bundle documentation about installation, testing, code quality checks, and miscellany about the package itself, not examples or code documentation.

File: {{ cookiecutter.package_name }}/docs/documentation/installation.rst (hunk 2 above)
Comment (nstarman) [reply {{ cookiecutter.package_name }}/docs/documentation/installation.rst:54]: I've thinking about this. Badges can be usefully integrated in all manner of places. Would it be useful to have a cookiecutter Q whether to insert badges? Then this just needs an if-statement.

File: {{ cookiecutter.package_name }}/docs/documentation/installation.rst (hunk 4 above)
Comment (nstarman) [reply {{ cookiecutter.package_name }}/docs/documentation/installation.rst:33]: Why delete these, doesn't it enable RST linking?

**Comment by pllim** [comment]
Personally I would advice to keep the template simple and not add (too many) things that a majority of the users might not need. Power users already know how to insert these stuff without the template.

Perhaps @Cadair or others have different opinions.

**Comment by pllim** [comment]
See https://github.com/astropy/package-template#deprecation-warning

**Pull request closed without merging**